integration/src/integration/java/pl/allegro/tech/hermes/integration/MessageBufferLoadingTest.java

```diff
@@ -70,7 +70,8 @@ public void tearDown() throws IOException {
         FileUtils.deleteDirectory(tempDir);
     }
 
-    @Test
+
+    @Test(enabled = false)
     public void shouldBackupMessage() throws Exception {
```

Comment:
Name of this test suggests that it is rather important. I think it would be nice to have it enabled.